installation/README.md

```diff
@@ -0,0 +1,93 @@
+# Installing Config Sync manually
+
+This document is a guide on how to manually install Config Sync on a Kubernetes
+cluster without Fleet Management.
+
+The recommended approach is to [install Config Sync managed by fleet](https://cloud.google.com/kubernetes-engine/enterprise/config-sync/docs/how-to/installing-config-sync).
+
+This document includes configurations for some common user journeys, but is not
+an exhaustive list of how a Config Sync installation can be customized.
+
+## Pre-requisites
+
+This guide requires the following command line tools:
+
+- [kubectl](https://kubernetes.io/docs/tasks/tools/)
+- [kustomize](https://github.com/kubernetes-sigs/kustomize)
+
+## Uninstall Config Management
+
+**Prerequisites**:
+- Set the kubectl context in your shell to the desired cluster before proceeding.
+- Ensure [hierarchy controller is disabled](https://cloud.google.com/kubernetes-engine/enterprise/config-sync/docs/how-to/migrate-hierarchy-controller#kubectl) before proceeding.
+
+
+If you have previously installed Config Management on your cluster, Config Management
+must first be uninstalled before following this guide to install/upgrade Config Sync.
+
+You can check if Config Management is installed on your cluster with the following command:
+
+```shell
+kubectl get configmanagement
+```
+
+If the output is not empty, Config Management is currently installed on the cluster
+and must be uninstalled before proceeding. To uninstall Config Management, there are
```

Comment:
I think users would want to know what the impact is when doing any "uninstallation". It might be easier to just talk about ConfigManagement Operator manifest per https://cloud.google.com/kubernetes-engine/enterprise/config-sync/docs/downloads

Thinking more about it, given that OSS installation doesn't really use the ConfigManagement Operator, this doc might be more suitable to live in GCP doc site (manual installation guide)

Reply:
Or maybe the OSS doc doesn't focus on removing ConfigManagement Operator, given that's part of manual installation in GCP

Reply:
Where do you suggest we host/publish the uninstall shell script?

Reply:
Also as a counter-argument we already have `nomos migrate` which knows how to remove `ConfigManagement` operator

Reply:
An example of publishing the script in docs is https://cloud.google.com/kubernetes-engine/enterprise/config-sync/docs/how-to/stopping-resuming-syncing#stopping-single

Reply:
Are you comfortable with not having any automated testing for the uninstall script and/or the end to end migration flow? One of the benefits of defining these in the repository here is for automated testing. That's my primary concern, otherwise I'm on board with moving this all to GCP docs.

Reply:
We can have the tests here. I think don't need to include it in the OSS docs. Given that it doesn't apply to existing OSS users, they might be confused.

Reply:
Just to clarify:
- The uninstall script will be defined in two places: the repo and the GCP docs
- The uninstall script will be in the repo only for the purpose of testing, and not published as a release artifact
- The GCP docs will reference the inline uninstall shell script and no references will be made to the in-repo script

Is that right?

Reply:
Sounds good. For details of what the GCP docs would look like, we'll need tech writer's input too and that would happen later.

Reply:
Ack, updated https://github.com/GoogleContainerTools/kpt-config-sync/pull/1456 accordingly